dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops

```diff
@@ -156,6 +156,10 @@ JSON_FIELDS_INVALID_CHARACHTER=TRAS3011W: An invalid character was provided in t
 JSON_FIELDS_INVALID_CHARACHTER.explanation= Only letters, numbers, and : are accepted in the JSON logging field name mapping property
 JSON_FIELDS_INVALID_CHARACHTER.useraction= Change your JSON logging field name mapping property value to remove invalid characters
 
+JSON_ACCESS_LOG_NO_HEADER_NAME_SPECIFIED=TRAS3012W: A header name is required to be specified for the logFormat field(s): {0}.
+JSON_ACCESS_LOG_NO_HEADER_NAME_SPECIFIED.explanation=There is no header name specified for the field.
```

Comment:
need to be more specific here that the JSON logging is using the logFormat attribute from accessLogging to determine which fields to include in the generated access log events, and that the logFormat is missing header names.

Reply:
I fixed the message but still not sure if it's clear enough